lib/extensions.js

```diff
@@ -0,0 +1,75 @@
+// Imports
+const fs = require('fs')
+const path = require('path')
+const foundationExtensions = require('../app/config').foundationExtensions || ['govuk-frontend']
+
+// Utils
+const pathFromRoot = (...all) => path.join.apply(null, [__dirname, '..'].concat(all))
+const pathToHookFile = packageName => pathFromRoot('node_modules', packageName, 'govuk-prototype-kit.config.json')
+const objectMap = (object, mapFn) => Object.keys(object).reduce((result, key) => {
+  result[key] = mapFn(object[key], key)
+  return result
+}, {})
+const removeDuplicates = arr => arr.filter((i, pos, self) => self.indexOf(i) === pos)
+const filterOutTreversesAndEmpty = part => part && part !== '..'
+const cloneArray = arr => arr.slice(0)
+
+// Reused Functions
+const lookupConfigForPackage = packageName => {
+  if (fs.existsSync(pathToHookFile(packageName))) {
+    return require(pathToHookFile(packageName))
+  } else {
+    return {}
+  }
+}
+
+const getPackageNamesInOrder = () => {
+  const dependencies = require(pathFromRoot('package.json')).dependencies || {}
+  const allNpmDependenciesInAlphabeticalOrder = Object.keys(dependencies).sort()
+  const installedTopPriorityDependencies = foundationExtensions
+    .filter(packageName => allNpmDependenciesInAlphabeticalOrder.indexOf(packageName) > -1)
```

Comment:
I think `.indexOf` can be replaced with `.includes`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

Reply:
Excellent.  Will do.